packages/payment-processor/test/payment/any-to-erc20-batch-proxy.test.ts

```diff
@@ -11,17 +11,28 @@ import { getErc20Balance } from '../../src/payment/erc20';
 import Utils from '@requestnetwork/utils';
 import { revokeErc20Approval } from '@requestnetwork/payment-processor/src/payment/utils';
 import { EnrichedRequest, IConversionPaymentSettings } from '../../src/index';
-import { currencyManager } from './shared';
+// import { currencyManager } from './shared';
 import {
   approveErc20BatchConversionIfNeeded,
   getBatchConversionProxyAddress,
   payBatchConversionProxyRequest,
   prepareBatchConversionPaymentTransaction,
 } from '../../src/payment/batch-conversion-proxy';
 import { batchConversionPaymentsArtifact } from '@requestnetwork/smart-contracts';
-import { UnsupportedCurrencyError } from '@requestnetwork/currency';
+import { CurrencyManager, UnsupportedCurrencyError } from '@requestnetwork/currency';
 import { BATCH_PAYMENT_NETWORK_ID } from '@requestnetwork/types/dist/payment-types';
 
+const currencyManager = new CurrencyManager([
+  ...CurrencyManager.getDefaultList(),
+  {
```

Comment:
why not adding that to the currency package for the supported tokens in network private as it's done for testnets?

Reply:
it breaks others' tests